### PR TITLE
Update Turbo Config

### PR DESCRIPTION
**Pull request by lishaduck**
I didn't realize Turbo automatically strips the env nowadays, so tests don't pick up the token. This should significantly reduce how often we get rate-limited in tests.

### CODE REVIEW

**Comment by lishaduck** [comment]
Note: CI is failing b/c CI is failing, it nothing to do with this PR. See #256.

File: turbo.json
Comment (lishaduck) [inline turbo.json:1]: I didn't format this, so prettier's complaining.
Will get to it... someday.

**Comment by lishaduck** [comment]
If we don't hit rate limits, this is good to go from me 👍

**Comment by jfmengels** [comment]
This looks really nice! Thank you :relaxed:

**Pull request merged**